### PR TITLE
contrib/waybar: new package (0.10.0)

### PR DESCRIPTION
**Pull request by anixedn**

### CODE REVIEW

File: contrib/waybar/template.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "libsigc++",
+    "libxkbregistry",
+    "playerctl-devel",
+    "spdlog-devel",
+    "udev-devel",
+    "upower-devel",
+    "wayland-devel",
+    "wireplumber-devel",
+]
+checkdepends = ["catch2-devel", "libdbusmenu-devel"]
```
Comment (nekopsykose) [inline contrib/waybar/template.py:35]: both of these in makedepends

File: contrib/waybar/template.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+depends = ["libdbusmenu-gtk3"]
```
Comment (nekopsykose) [inline contrib/waybar/template.py:36]:
```suggestion
```

this is already traced

File: contrib/waybar/template.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+build_style = "meson"
+configure_args = [
+    "-Dcava=disabled",
+    "-Ddbusmenu-gtk=enabled",
+    "-Dlibcxx=true",
+    "-Dmpd=enabled",
+    "-Dsystemd=disabled",
+    "-Dtests=enabled",
+]
+hostmakedepends = ["cmake", "meson", "pkgconf", "scdoc", "wayland-protocols"]
```
Comment (q66) [inline contrib/waybar/template.py:13]: cmake is not a dependency

File: contrib/waybar/template.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+makedepends = [
+    "catch2-devel",
+    "clang-tools-extra",
```
Comment (q66) [inline contrib/waybar/template.py:16]: not needed

File: contrib/waybar/template.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    "fmt-devel",
+    "gobject-introspection",
```
Comment (q66) [inline contrib/waybar/template.py:18]: this is a host dependency

File: contrib/waybar/template.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    "gtk-layer-shell-devel",
+    "gtkmm3.0-devel",
+    "jsoncpp-devel",
+    "libdbusmenu-devel",
+    "libevdev-devel",
+    "libgirepository-devel",
+    "libinput-devel",
+    "libmpdclient-devel",
+    "libnl-devel",
+    "libsigc++",
```
Comment (q66) [inline contrib/waybar/template.py:28]: libsigc++2-devel

File: contrib/waybar/template.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    "wireplumber-devel",
+]
+pkgdesc = "Wayland bar for Sway and Wlroots based compositors"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "MIT"
+url = "https://github.com/Alexays/Waybar"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "3af6665889868f2334ba1793c8b0f3104c4c3b176a8c759f0d08f07266ad2620"
+# breaks tests
+hardening = ["!vis"]
```
Comment (q66) [inline contrib/waybar/template.py:44]: this is the default, no need to set

File: contrib/waybar/template.py [hunk 8]
```diff
@@ -0,0 +1,48 @@
+pkgname = "waybar"
+pkgver = "0.10.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dcava=disabled",
+    "-Ddbusmenu-gtk=enabled",
+    "-Dlibcxx=true",
```
Comment (q66) [inline contrib/waybar/template.py:8]: this only adds some extra cflags that we don't need

File: contrib/waybar/template.py [hunk 9]
```diff
@@ -0,0 +1,48 @@
+pkgname = "waybar"
+pkgver = "0.10.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
```
Comment (q66) [inline contrib/waybar/template.py:5]: list out all the features explicitly

File: contrib/waybar/template.py [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    "gtkmm3.0-devel",
+    "jsoncpp-devel",
+    "libdbusmenu-devel",
+    "libevdev-devel",
+    "libgirepository-devel",
+    "libinput-devel",
+    "libmpdclient-devel",
+    "libnl-devel",
+    "libsigc++",
+    "libxkbregistry",
```
Comment (q66) [inline contrib/waybar/template.py:29]: libxkbcommon-devel

File: contrib/waybar/template.py [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+    "libsigc++2-devel",
+    "libxkbcommon-devel",
+    "playerctl-devel",
+    "spdlog-devel",
+    "udev-devel",
+    "upower-devel",
+    "wayland-devel",
+    "wireplumber-devel",
+]
+pkgdesc = "Wayland bar for Sway and Wlroots based compositors"
```
Comment (q66) [inline contrib/waybar/template.py:54]: wlroots-based

File: contrib/waybar/template.py [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+    "jsoncpp-devel",
+    "libdbusmenu-devel",
+    "libevdev-devel",
+    "libgirepository-devel",
+    "libinput-devel",
+    "libmpdclient-devel",
+    "libnl-devel",
+    "libsigc++2-devel",
+    "libxkbcommon-devel",
+    "playerctl-devel",
```
Comment (q66) [inline contrib/waybar/template.py:47]: probably want explicit pipewire-devel as well

File: contrib/waybar/patches/update-wireplumber-module.patch [hunk 13]
```diff
@@ -0,0 +1,373 @@
+diff --git a/src/modules/wireplumber.cpp b/src/modules/wireplumber.cpp
```
Comment (nekopsykose) [inline contrib/waybar/patches/update-wireplumber-module.patch:1]: all three of these patches should be one patch, just https://github.com/Alexays/Waybar/commit/2326727ccbf0456ccfd631e748955f7f67c44a4e.patch

**Pull request merged**